framework/src/modules/nft/types.ts

```diff
@@ -58,6 +58,12 @@ export interface NFTAttributes {
 }
 
 export interface NFT {
+	owner: Buffer;
+	attributesArray: NFTAttributes[];
+	lockingModule?: string;
+}
+
+export interface NFTOutputEndpoint {
```

Comment:
Instead of creating a duplicated interface with same properties but different types, I think it would be better to follow the pattern which we've been [using throughout the codebase](https://github.com/LiskHQ/lisk-sdk/blob/a251bb369e61a54105c3099138768c814af65713/elements/lisk-api-client/src/types.ts#L177-L186):

```typescript
export type JSONObject<T> = Replaced<T, bigint | Buffer, string>;

export type NFTJSON = JSONObject<NFT>;
```

Reply:
valid point 👍 I updated it